src/styles/index.scss

```
@@ -1,3 +1,147 @@
-body {
+// #region page
+@import "./variables";
+@import "https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap";
+
+* {
   margin: 0;
+  padding: 0;
 }
+
+// #endregion
+// #region container
+.card_container {
+  padding: 32px 16px 16px;
+  width: 200px;
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  border-radius: 5px;
+  border: 1px solid #F3F3F3;
+}
+
+// #endregion
+// #region image, title and subtitle
+.card_product-image {
+  width: 160px;
+  height: 134px;
+  background-image: url(../images/imac.jpeg);
+  background-size: 100% 100%;
+  background-repeat: no-repeat;
+  background-position: center;
+  margin-bottom: 40px;
+}
+
+.card_product-name {
+  margin: 0 0 4px;
```

use only margin-bottom
```suggestion
  margin-bottom: 4px;
```